.nuget/create_nuget.py

```diff
@@ -36,9 +40,47 @@
         <dependencies> </dependencies>
     </metadata>
         <files>
+        <file src="build\**" target="build"/>
         <file src="runtimes\**" target="runtimes"/>
         </files>
     </package>
     ''' % version
+    prop_str = '''
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup Condition="Exists('packages.config') OR
+                            Exists('$(MSBuildProjectName).packages.config') OR
+                            Exists('packages.$(MSBuildProjectName).config')">
+        <Content Include="$(MSBuildThisFileDirectory)/../runtimes/win-x64/native/*.dll"
+                Condition="'$(PlatformTarget)' == 'x64'">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        <Visible>false</Visible>
+        </Content>
+        <Content Include="$(MSBuildThisFileDirectory)/../runtimes/win-x64/native/*.exe"
+                Condition="'$(PlatformTarget)' == 'x64'">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        <Visible>false</Visible>
+        </Content>
+    </ItemGroup>
+    </Project>
+    '''
+    target_str = '''
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <EnableMLUnsupportedPlatformTargetCheck Condition="'$(EnableMLUnsupportedPlatformTargetCheck)' == ''">true</EnableMLUnsupportedPlatformTargetCheck>
```

Comment:
I think we should use a different MSBuild property name here.  `EnableMLUnsupportedPlatformTargetCheck` should be used for ML.NET (since it is "Enable ML Unsupported...")